buildSrc/src/main/resources/de/thetaphi/forbiddenapis/gradle/plugin-init.groovy

```diff
@@ -0,0 +1,60 @@
+/*
+ * (C) Copyright Uwe Schindler (Generics Policeman) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Initializes the plugin and binds it to project lifecycle. */
+/** This is a workaround for https://github.com/policeman-tools/forbidden-apis/issues/189 */
+
+import org.gradle.api.plugins.JavaBasePlugin;
```

Comment:
@uschindler sorry for dragging you in (again!), found a workaround by subclassing `CheckForbiddenApis` 

```
public class CheckForbiddenApis extends de.thetaphi.forbiddenapis.gradle.CheckForbiddenApis {
    @Override
    @InputFiles
    @IgnoreEmptyDirectories
    @SkipWhenEmpty
    public FileTree getClassFiles() {
        return super.getClassFiles();
    }
}
```

and overriding the `plugin-init.groovy` with this task class, this is obviously not a fix but unblocks 7.4.x migration (at least for OpenSearch), do you have an opinion on that? Thank you!